lib/utils/debounce.js

```diff
@@ -45,10 +46,19 @@ export class Debouncer {
    */
   cancel() {
     if (this.isActive()) {
-      this._asyncModule.cancel(/** @type {number} */(this._timer));
+      this._cancelAsync();
       this._timer = null;
+      debouncerQueue.delete(this);
```

Comment:
Add comment about what this does to flush ordering.

Reply:
Done